personalization-webcomponents/src/components/ChecklistHeader.vue

```diff
@@ -0,0 +1,43 @@
+<template>
+  <muc-intro tagline="Checkliste" :title="checklist.title" :img="getChecklistIconByTitle(checklist.title)" :imgAlt="checklist.title">
+    <p style="padding-bottom: 18px">
+      <b>Erstellungsdatum: </b>{{ props.checklist.lastUpdated.toLocaleString() }}
+    </p>
+    <p>
+      <b>Aufgaben: </b>
+      <muc-chip
+          v-if="todoCount"
+          style="margin-right: 16px"
+          background-color="#FDD1AC"
+      >
+        {{ todoCount }} offen
```

Comment:
You are currently only displaying the number of open items but the number of "done" items should be displayed too, if available.

<img width="372" height="87" alt="Image" src="https://github.com/user-attachments/assets/e28fc4b7-8221-4f04-8828-216db735cece" />